src/blocks/scratch3_motion.js

```diff
@@ -56,8 +56,10 @@ Scratch3MotionBlocks.prototype.goTo = function (args, util) {
         targetX = util.ioQuery('mouse', 'getX');
         targetY = util.ioQuery('mouse', 'getY');
     } else if (args.TO === '_random_') {
-        targetX = this.runtime.constructor.STAGE_WIDTH * (Math.random() - 0.5)
-        targetY = this.runtime.constructor.STAGE_HEIGHT * (Math.random() - 0.5)
+        var stageWidth = this.runtime.constructor.STAGE_WIDTH;
+        var stageHeight = this.runtime.constructor.STAGE_HEIGHT;
```

Comment:
I'm fine with this, alternatively if you're worried about it I guess we could add `runtime.getStageDimensions()`